Mathlib/RingTheory/SimpleModule.lean

```diff
@@ -148,6 +240,77 @@ lemma isSemisimpleModule_of_isSemisimpleModule_submodule' {p : ι → Submodule
     IsSemisimpleModule R M :=
   isSemisimpleModule_of_isSemisimpleModule_submodule (s := Set.univ) (fun i _ ↦ hp i) (by simpa)
 
+theorem IsSemisimpleModule.sup {p q : Submodule R M}
```

Comment:
While you're at it, add IsSemisimpleModule.sSup and .iSup?

Reply:
There's already the biSup version (from which this is deduced) and it's easy to convert iSup and sSup using [iSup_univ](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/CompleteLattice.html#iSup_univ) and [sSup_eq_iSup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/CompleteLattice.html#sSup_eq_iSup), so I think we can omit the sSup and iSup versions.

Reply:
OK, then…